zilliqa/src/api/trace.rs

```diff
@@ -4,14 +4,23 @@ use std::{
 };
 
 use alloy_primitives::B256;
-use alloy_rpc_types_trace::parity::{TraceResults, TraceType};
+use alloy_rpc_types_trace::{
+    geth::{GethDebugTracingOptions, TraceResult},
+    parity::{TraceResults, TraceType},
+};
 use anyhow::Result;
 use jsonrpsee::{types::Params, RpcModule};
 
-use crate::{crypto::Hash, node::Node};
+use crate::{crypto::Hash, message::BlockNumber, node::Node};
 
 pub fn rpc_module(node: Arc<Mutex<Node>>) -> RpcModule<Arc<Mutex<Node>>> {
-    super::declare_module!(node, [("trace_replayTransaction", replay_transaction)])
+    super::declare_module!(
+        node,
+        [
+            ("trace_replayTransaction", replay_transaction),
+            ("debug_traceBlockByNumber", debug_trace_block_by_number),
```

Comment:
It'd be neater to have two modules - one for trace_ and one for debug_, but probably only if they are both going to acquire lots of API calls. If there are only a few, I guess this will have to do (the problem is that it means I'll have to introduce extra complexity in the doc generator to reformat API calls by prefix with a bunch of heuristics - because ZIL API calls have no prefix .. )